roles/elasticsearch/defaults/main.yml

```diff
@@ -19,3 +19,5 @@ elasticsearch_config:
   http.cors.allow-origin: '/.*/'
   network.host: '{{ elasticsearch_interface }}'
   http.port: '{{ elasticsearch_port }}'
+
+java_package_name: java
```

Comment:
Do we have anything else that depends on java?  If so, it would make sense to put this in a separate role.  Otherwise, I guess I don't have a strong opinion.